report/sql_tasks/tasks/report/create_from_scratch/00_schema/02_create_fdw_server.jinja2.sql

```diff
@@ -14,7 +14,7 @@ CREATE EXTENSION IF NOT EXISTS postgres_fdw;
         );
 
     {% for user in users %}
-        DROP USER MAPPING IF EXISTS FOR "{{user}}" SERVER "h_us_server";
```

Comment:
h_us_server was very deleted

Reply:
It's the only way to be sure :mushroom: :cloud: